java/compaction/compaction-core/src/main/java/sleeper/compaction/strategy/ShouldCreateJobsStrategy.java

```diff
@@ -27,9 +26,9 @@ public interface ShouldCreateJobsStrategy {
     default void init(InstanceProperties instanceProperties, TableProperties tableProperties) {
     }
 
-    long maxCompactionJobsToCreate(Partition partition, List<FileReference> activeFilesWithJobId, List<FileReference> activeFilesWithNoJobId);
+    long maxCompactionJobsToCreate(String partitionId, List<FileReference> filesWithJobId);
```

Comment:
I can't see from the name of the second parameter which files I should expect to be there.